packages/embeds/js/src/features/blocks/bubbles/video/components/VideoBubble.tsx

```diff
@@ -77,9 +77,19 @@ export const VideoBubble = (props: Props) => {
               }
             >
               <video
-                autoplay={props.onTransitionEnd ? false : true}
+                autoplay={
+                  props.onTransitionEnd
+                    ? !(
+                        props.content?.areControlsDisplayed ??
+                        defaultVideoBubbleContent.areControlsDisplayed
+                      )
+                    : false
+                }
```

Comment:
I am not sure if this is correct

I took logic from `AudioBubble.tsx`

https://github.com/baptisteArno/typebot.io/blob/e755f08b403cdb4193ce93b1bb02098444f99a89/packages/embeds/js/src/features/blocks/bubbles/audio/components/AudioBubble.tsx#L61-L66